packages/typescript-private/console-host-agent/src/http-specs/private-device.ts

```diff
@@ -35,6 +35,9 @@ export class CreateDeviceRequestBody implements Pick<Required<Device>, 'serial'
   @IsNumber()
   @IsNotEmpty()
   isVirtual!: number;
+
+  @IsString()
+  memory!: string;
```

Comment:
Does it need nullable or `IsNotEmpty()` here?

Reply:
@Ikalli You're right. But I didn't implement the memory of the Macos device. 
<img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Smilies/Beaming%20Face%20with%20Smiling%20Eyes.png" alt="Beaming Face with Smiling Eyes" width="25" height="25" />